Cargo.toml

```diff
@@ -167,6 +167,7 @@ locktick = [
   "snarkos-node-sync/locktick",
   "snarkos-node-tcp/locktick"
 ]
+serial = [ "snarkos-cli/serial" ]
```

Comment:
Should we just let this also enable `serial` in snarkVM (for easy use in flamegraphing, debugging, and anything else really). Do we have any usecase for `serial` in snarkOS but not in snarkVM? As a developer, we can always adjust it, its just about what we'd like to have as default.

CC @niklaslong , this also applies to other services depending on snarkVM